Jellyfin.Networking/Manager/NetworkManager.cs

```diff
@@ -1123,5 +1130,79 @@ private void PrintNetworkInformation(NetworkConfiguration config, bool debug = t
                 _logger.Log(logLevel, "Filter list: {0}", _remoteAddressFilter.Select(s => s.Prefix + "/" + s.PrefixLength));
             }
         }
+
+        /// <summary>
+        /// Initializes the LAN IP address that the server advertises for cast receivers.
+        /// A LAN device like a Chromecast receiver needs an unique LAN address of the Jellyfin server.
+        /// For IPv4, a Private network address (like 192.168.0.0/16 )serves this purpose, and for IPv6
+        /// the Unique Local Address (ULA, fd00::/8) does. An IPv6 global address (2000::/3) also works
+        /// as unique server address but is ranked as less desirable than a local address.
+        /// See e.g. https://en.wikipedia.org/wiki/Private_network
+        ///
+        /// If multiple addresses are available:
+        /// - prefer Unique Local Addresses (ULA) or Site Local Addresses over other (global) addresses.
+        /// - all things equal, prefer IPv6 over IPv4 as the former is more state-of-the-art.
```

Comment:
IPv6 doesn't always work and I have seen reports that CC doesn't play well with it, so we should honestly prefer IPv4 for this

Reply:
Ok, this also worked fine on my system but I don't have much experience with others. I changed the preferred ranking.